tests/e2e/specs/dashboard/notifications.test.js

```diff
@@ -69,7 +69,8 @@ describe( 'core site notifications', () => {
 
 			// Ensure the notification is displayed.
 			await page.waitForSelector(
-				`#${ testSiteNotification.id }.googlesitekit-publisher-win--is-open`
+				`#${ testSiteNotification.id }.googlesitekit-publisher-win--is-open`,
+				{ timeout: 10_000 } // Core site notifications are delayed 5s for surveys.
```

Comment:
See 
* https://github.com/google/site-kit-wp/issues/3508

This broke due to reducing the default timeout to 5s in the initial PR which should be more than enough time in most cases, so we can treat this as an exception rather than increase the default.